$MirOS: ports/lang/nwcc/patches/patch-cc_main_c,v 1.1 2008/10/30 17:11:15 tg Exp $
--- cc_main.c.orig	Wed Mar  4 16:11:42 2009
+++ cc_main.c	Fri Apr 17 14:23:47 2009
@@ -714,7 +714,7 @@ main(int argc, char *argv[]) {
 	sprintf(asm_flags, "-arch %s -force_cpusubtype_ALL ",
 		archflag == ARCH_X86? "i386": "x86_64");
 #  else
-#     error "Unsupported system (currently only Linux/FreeBSD/OpenBSD on x86 \
+#     error "Unsupported system (currently only FreeBSD/Linux/MidnightBSD/MirBSD/OpenBSD on x86 \
 are supported)"
 #  endif
 #elif defined __amd64__ || defined __x86_64__
